bin/yaml/libraries.yaml

```diff
@@ -354,6 +354,17 @@ libraries:
       target_prefix: V
       targets:
         - 0.6.3
+    xerces-c:
+      type: github
+      repo: apache/xerces-c
+      check_file: README
+      target_prefix: v
+      targets:
+        - 3.2.3
+      build_type: cmake
+      lib_type: static
+      extra_cmake_arg:
+        - -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=OFF -Dxmlch-type=uint16_t
```

Comment:
Can you leave the build_type out. The buildprocess sets the buildtype automatically to Debug.

Reply:
Done e2751d46344f64f7e63b65f6b183fc7c2f3989b2